Warhammer 3rd dice roller

silentmark opened the pull request:
Hi, 

I would like to contribute with wfrp 3rd edition special dice roller :-)

Commit:
Warhammer 3rd dice roller

Commit:
adding images

Commit:
missed interface declaration.

BernhardPosselt commented:
Looks very good, thank you! Have you checked the license for the images?

silentmark commented:
Honestly, i'm not sure about the licenses - i reused the images found for roll20 script, from here: https://app.roll20.net/forum/post/1756666/warhammer-fantasy-roleplay-3rd-edition-dice-script-help/?pageforid=1938504#post-1938504

Commit:
updated the images and gifs for transparency

BernhardPosselt commented:
Yeah, sorry, can't merge this pull request then :(

BernhardPosselt commented:
Can you check if you can find images that are compatible with the AGPL?

Commit:
fixed issue with double bane returned only 1 bane

silentmark commented:
Sorry, i've missed your comment, i will try to look for something and update PR

illotum commented:
Would it help if I traced the images into svg?

BernhardPosselt commented:
@illotum if they have a distinct design compared to the original, maybe that's fine.

illotum commented:
Several game-icons.net assets, all my work:
[wfrp3-faces-CC-BY-3.0.zip](https://github.com/BernhardPosselt/foundryvtt-special-dice-roller/files/5235394/wfrp3-faces-CC-BY-3.0.zip)
Edit. Realized you need the sigils: 
[wfrp3-sigils-CC-BY-3.0.zip](https://github.com/BernhardPosselt/foundryvtt-special-dice-roller/files/5236398/wfrp3-sigils-CC-BY-3.0.zip)

Commit:
gitignore

Commit:
merge

Commit:
updated imges for WFRP thanks to illotum

silentmark commented:
@illotum , thank you, i have added your images inplace the ones found somewhere on roll and added you in Readme. @BernhardPosselt , if you find some time, please check if it's compliant.

BernhardPosselt commented:
Thanks!
